lib/components/Snackbar/snackbar.tsx

```diff
@@ -34,16 +36,17 @@ export const Snackbar = ({
     };
 
     useEffect(() => {
-        timerRef.current = snackbarDelayedRemove(id);
+        id && (timerRef.current = snackbarDelayedRemove(id));
 
         return () => {
             clearTimeout(timerRef.current ?? "");
         };
     }, [message]);
 
-    const handleClose = (delay: number = 100) => {
+    const handleClose = () => {
+        onCloseAction?.();
         if (timerRef) clearTimeout(timerRef.current ?? "");
-        snackbarDelayedRemove(id, delay);
+        id && snackbarDelayedRemove(id, 100);
```

Comment:
why aren't we using the `delay` anymore? We are allowing the consumer to add their own delay while we are setting a default delay of `100`

Reply:
Because I noticed delay is not passing into the function then I remove it. I'll add this as a props for consumer